device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x100G/0/jr2cp-nokia-18x100g-4x25g-config.bcm

```diff
@@ -1477,6 +1477,7 @@ protocol_traps_mode.BCM8885X=IN_LIF
 rate_ext_mdio_divisor=16
 schan_intr_enable.BCM8885X=0
 schan_timeout_usec.BCM8885X=900000
+system_ref_core_clock_khz.BCM8885X=1600000
```

Comment:
i think we only need to update on fabric side ?

Reply:
> i think we only need to update on fabric side ?

@vmittal-msft , We were using default value which is 1.6GHZ and we are explicitly setting the same value for J2C+ now.

Reply:
Thanks. @saksarav-nokia Could you run whole test suite as well as some level of ixia testing to make sure this is not causing any new issues ?